source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -17,31 +17,39 @@ The global vulnerability database is created automatically, currently pulling da
+The results are presented as alerts and also stored in a per-agent vulnerabilities inventory. So you can check the last scan alerts, or query every single agent's vulnerabilities inventory.
 
 .. _vuln_det_scan_types:
 
 Scan types
 ^^^^^^^^^^
 
-The Vulnerability Detector module can run a scan on startup (:ref:`run_on_start <vuln_det_run_on_start>`) and every certain period of time (:ref:`interval <vuln_det_interval>`).
-In any of these cases, the packages and the operating system that were already scanned will be re-scanned if the database of vulnerabilities receives new CVEs information and :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires.
-This improves the performance and avoids repeated alerts during a configurable time. We have then three different types of scan:
+It can run a scan on startup (:ref:`run_on_start <vuln_det_run_on_start>`) and every certain period of time (:ref:`interval <vuln_det_interval>`).
+In any of these cases, the packages and the operating system that were already scanned will not be re-scanned, only the new ones will do and alerts are going to be generated.
+If the database of vulnerabilities receives new CVEs information and :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires all packages, as well as the operating system, will be re-scanned.
 
-- Baseline: This scan type will be triggered the first time after Vulnerability Detector is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
-- Full scan: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the newly detected vulnerabilities and the ones that no longer affect the agent will be alerted.
-- Partial scans: Only new packages are scanned and the vulnerabilities detected are alerted.
+We have then three different types of scan:
+
+- `Baseline`: This scan type will be triggered the first time after `Vulnerability Detector` is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
+- `Full scan`: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the vulnerabilities update will be alerted.
+- `Partial scan`: Only new packages are scanned and, as it is in `Full scans`, any vulnerability update will be reported.
```

Comment:
In my previous comment, I suggested adding the same explanation "as it is in `Full scans`", not to add this expression. I think is not clear. We should only explain that 
`Only new packages are scanned. As a result, the vulnerabilities update will be alerted.`

Reply:
Yes, I understood, I just thought it would be good to add that sentence as it is. Fixed now.